SConstruct

```diff
@@ -340,6 +340,16 @@ def scons(): # pylint: disable=too-many-locals
 
     env = Environment(TOOLS=['extra', 'default', 'textfile'])
 
+    # Scons strips out the environment, however to be able to build daos using the interception
+    # library we need to add a few things back in.
+    if 'LD_PRELOAD' in os.environ:
+        env['ENV']['LD_PRELOAD'] = os.environ['LD_PRELOAD']
```

Comment:
(pylint-invalid-sequence-index) Sequence index is not an int, slice, or instance with __index__